Pod/Classes/Controllers/SoundCloudLoginWebViewController.m

```diff
@@ -16,7 +16,7 @@ @interface SoundCloudLoginWebViewController () <UIWebViewDelegate>
 @implementation SoundCloudLoginWebViewController
 
 + (UINavigationController *)instantiateWithLoginURL:(NSString *)loginURL redirectURL:(NSString *)redirectURL resultBlock:(void (^)(BOOL result, NSString *code))resultBlock {
-    UINavigationController *navController = [[UIStoryboard storyboardWithName:@"SoundCloudLogin" bundle:nil] instantiateInitialViewController];
+    UINavigationController *navController = [[UIStoryboard storyboardWithName:@"SoundCloudLogin" bundle:[NSBundle bundleForClass:self]] instantiateInitialViewController];
```

Comment:
Thanks for the PR! I just got one doubt: Is there any difference between using `[NSBundle bundleForClass:self]`or `[NSBundle bundleForClass:[self class]]` ?